### PR TITLE
Support undertow 1.x

### PR DESCRIPTION
**Pull request by langyan1022**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
issues #3634
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- undertow 1.x plugin
   Support undertow 1.x
   It should be noted that 1.x plugin and 2.x plugin cannot coexist.(1.x版本和2.x版本不能共存，保留一个即可)

### CODE REVIEW

**Comment by langyan1022** [comment]
拷贝skywalking 6.0.0 的undertow-2.x-plugin 插件，仅仅改了包名

在skywalking 6.5.0-SNAPSHOT版本 undertow 1.4.27版本测试通过

File: apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+undertow 1.x 插件
+拷贝skywalking 6.0.0 的undertow-2.x-plugin 插件
+
+在skywalking 6.5.0-SNAPSHOT版本 undertow 1.4.27版本测试通过
```
Comment (JaredTan95) [inline apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md]: English Please.

Comment (langyan1022) [reply apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md]: ok，already edited

**Comment by dmsolr** [comment]
Hi @langyan1022 , I think we don't need to copy that. Maybe we rename the directory only. And add related versions list into `support-version.list`(testcase.yml). 

Also, the [plugin testcases repo](https://github.com/SkyAPMTest/agent-auto-integration-testcases) is closed. If you are interested that, welcome to(need to :-) ) join us what we have an [immigration plan](https://github.com/apache/skywalking/issues/3583).

File: apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+### undertow 1.x plugin
+Copy the undertow-2.x-plugin plugin from skywalking 6.0.0
+
+Passed the test in skywalking 6.5.0-SNAPSHOT version and undertow 1.4.27 version
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/undertow-plugins/undertow-1.x-plugin/readme.md]: This document is strange, why do we need this? If we accept a plugin for new version, you need to change the existing one. And this plugin must pass the tests for multiple versions.

Remove this file, please. And submit another pr to move undertow plugin test into this repo. 
Last step, make the existing old plugin passed 1.x versions. Notice, onr version is not enough.

**Comment by wu-sheng** [comment]
Why 1.x and 2.x can exist together? What is the conflict?

**Comment by langyan1022** [comment]
> Why 1.x and 2.x can exist together? What is the conflict?



> Why 1.x and 2.x can exist together? What is the conflict?

I thought undertow2.x means only undertow 2.0 or later. And the existing undertow2.x plugin does not support or have problems in the 1.x version of undertow, sorry. Maybe I understand it wrong.

**Comment by wu-sheng** [comment]
The name means it is tested and designed in those versions. We don't know the status of 1.x. If it could be compatible, the name could be changed.
But even in that, plugin test immigration pr is required before this one.
